Cargo.toml

```diff
@@ -12,6 +12,7 @@ description = "Cross platform scripting for deno task"
 [dependencies]
 anyhow = "1.0.55"
 futures = "0.3.16"
+path-dedot = "3.0.14"
```

Comment:
I tried path-clean, but it seemed to not be as good.

Reply:
Nice you found this one